macros/blueprint-proc-macro/src/report.rs

```diff
@@ -74,7 +75,7 @@ pub(crate) fn report_impl(args: &ReportArgs, input: &ItemFn) -> syn::Result<Toke
     let report_def = ReportDefinition {
         metadata: ReportMetadata {
             name: fn_name_string.clone().into(),
-            description: None, // TODO: Add support for description
+            description: args.description.as_ref().map(|r| r.as_str().into()),
```

Comment:
This one actually was not a TODO. This field will get updated by `blueprint-metdata` crate during build time. Hence, if we added this one here, it will be overwritten by the build step; I recommend leaving it with `None` and just add a comment explaining why.